src/services/completions.ts

```diff
@@ -753,6 +754,7 @@ namespace ts.Completions {
         log("getCompletionData: Is inside comment: " + (timestamp() - start));
 
         let insideJsDocTagTypeExpression = false;
+        let isInSnippetScope = false;

```

Comment:
should not this default to `true` then? it used to default to `Global` before. we do not want to change the behavior really since we have no test coverage here.

Reply:
I think this is correct -- we only assigned `completionKind = CompletionKind.Global;` in one place before, and that location is now the only assignment to `isInSnippetScope`.